api/observability/tracing/ssh/ssh.go

```diff
@@ -147,7 +143,7 @@ func Dial(ctx context.Context, network, addr string, config *ssh.ClientConfig, o
 // properly over the ssh connection.
 func NewClientConn(ctx context.Context, conn net.Conn, addr string, config *ssh.ClientConfig, opts ...tracing.Option) (ssh.Conn, <-chan ssh.NewChannel, <-chan *ssh.Request, error) {
 	tracer := tracing.NewConfig(opts).TracerProvider.Tracer(instrumentationName)
-	ctx, span := tracer.Start(
+	_, span := tracer.Start(
```

Comment:
Nit: I'd be tempted to keep this `ctx` shadowing. If someone edits the function in future, they'll almost certainly forgot to add this shadow back in and the trace won't be passed.

Reply:
Done 👍